src/plugin/file/kml/kmlnodelayerui.js

```diff
@@ -1092,9 +1092,13 @@ plugin.file.kml.KMLNodeLayerUICtrl.prototype.isFeatureFillable = function() {
   var features = this.getFeatures();
   var feature = features.length > 0 ? features[0] : null;
   if (feature) {
-    var geometry = feature.getGeometry();
+    const hasPolyGeom = [];
 
-    return os.geo.isGeometryPolygonal(geometry);
+    os.feature.forEachGeometry(feature, (geom) => {
+      hasPolyGeom.push(os.geo.isGeometryPolygonal(geom, true));
+    });
+
+    return hasPolyGeom.some((bool) => bool);
```

Comment:
It's probably cleaner to do this with `hasPolyGeom` as a boolean. In a normal `for` loop you could simply `break` once it becomes true, but with `forEachGeometry`, you'll have to do the whole loop no matter what, so you'll only ever want to set the flag once.

Reply:
Didn't even think of that, but makes sense